tests/infra/ccf.py

```diff
@@ -172,8 +172,8 @@ def start_and_join(self, args):
         cmd = ["rm", "-f"] + glob("member*.pem")
         infra.proc.ccall(*cmd)
 
-        self.consortium = infra.consortium.Consortium([1, 2, 3])
-        self.initial_users = [1, 2, 3]
+        self.consortium = infra.consortium.Consortium([0, 1, 2])
```

Comment:
Previously we created a `member1_cert.pem` and `member1_privk.pem`, and these were associated with the member with ID `0` inside CCF. This changes the default member (and user) names used locally to match their eventual CCF IDs. This results in all of the `-1` changes in `memberclient.py`. In general we should probably try to break the assumption that these IDs are in-sync to avoid domino bugs - maybe the default users should be `alice`, `bob`, and `charlie`?